release-calendar/src/client/stylesheets/channelTable.scss

```diff
@@ -0,0 +1,91 @@
+
+button:focus {
+    outline: none !important;
+}
+
+body {
+    font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
+    font-size: 14px;
+    line-height: 3em;
+    color: #24292e;
+}
+
+.rtv-button{
+    -webkit-appearance: none;
+    float:left;
+    width: 35%;
+    font-size: 14px;
+    height: 3em;
+    text-align: center;
+    padding-left: 0px;
+    padding-right: 0px;
+    background-color: white;
+}
+
+.label{
+    width: inherit;
+    margin-left: 4em;
+    padding-right: 0px;
+    text-align: left;
+    position: absolute;
+    height:3em;
+    text-overflow: clip;
+    overflow: hidden;
+    white-space: nowrap;
+    line-height:3em;
+    -webkit-appearance: button;
+}
+
+.row {
+    float:left;
+    padding-left: 0px;
+    padding-right: 0px;
+    padding-top: 0px;
+    width:65%;
+    height: 3em;
+    cursor: pointer;
+    user-select: none;
+    -webkit-appearance: button;
+    box-sizing: border-box;
+    border-width: 1px;
+    border-style: solid;
+    border-color: rgb(216, 216, 216) rgb(209, 209, 209) rgb(186, 186, 186);
+    border-image: initial;
+  }
+  
+/* Hide the browser's default*/
+.row input {
+  opacity: 0;
+  cursor: pointer;
+  float:left;
+  width: 0;
+  height: 0;
+}
+  
+.square {
+   float:left;
+   width: 3em;
+   height: 3em;
+   opacity: 100%;
+   background-color: black;
+}
+  
+/* On mouse-over*/
+.row:hover input ~ .square {
+    opacity: 50% !important;
+}
```

Comment:
What's going on here? I don't see any changes on hover.

Reply:
Problems were because I had changed that name for the classNames from the generic .square to more specific .square-lts. I changed this area pretty dramatically with mixins, should be better now.